frontend/src/booking/calendar.vue

```diff
@@ -137,24 +149,26 @@
         'setSelectedExam',
         'toggleBookingModal',
         'toggleCalendarControls',
+        'toggleOtherBookingModal',
         'toggleScheduling',
+        'toggleSchedulingIndicator',
+        'toggleSchedulingOther'
       ]),
       agendaDay() {
         this.$refs.bookingcal.fireMethod('changeView', 'agendaDay')
-        this.$refs.bookingcal.fireMethod('option', 'groupByResource', true)
       },
       agendaWeek() {
         this.$refs.bookingcal.fireMethod('changeView', 'agendaWeek')
-        this.$refs.bookingcal.fireMethod('option', 'groupByResource', true)
       },
       cancel() {
-        this.toggleScheduling(false)
-        this.navigationVisible(true)
-        this.toggleCalendarControls(true)
-        this.setSelectedExam(null)
+        this.finishBooking()
+        this.unselect()
       },
       initialize() {
         this.initializeAgenda().then( rooms => {
+          rooms.push(
```

Comment:
Does this belong here? This kinda seems like a test thing, or should be in the store instead of a create/mount lifecycle phase.